README.md

```diff
@@ -116,8 +116,10 @@ async function whatEverFunction() {
   const result = await vueScanner.scan();
 }
 
-
 ```
+_Note: For a **Nuxt** project, please ensure that you have a `.Nuxt` folder before scanning._
```

Comment:
@ravut8 Please change `.Nuxt` to `.nuxt`

Reply:
Oh, my bad. Just fixed.